scripts/project-setup-scripts/add-folder-roles-for-service-accounts-and-terra-billing-and-test-users.sh

```diff
@@ -30,7 +30,7 @@ else
     FOLDER_ID=950154083315 # folder: test.firecloud.org/tools (folder for dev FIABs)
     vault_env=dev
   elif [[ "${ENV}" == "fiab-qa" ]]; then
-    FOLDER_ID="TBD" # folder: quality.firecloud.org/qa # todo: to be created as part of https://broadworkbench.atlassian.net/browse/CA-1194 after completion of https://broadworkbench.atlassian.net/browse/DDO-1089
+    FOLDER_ID="147754536561" # folder: quality.firecloud.org/quality (folder for QA FIABs)
```

Comment:
IDs are from: https://broadworkbench.atlassian.net/browse/DDO-1089?focusedCommentId=44964

Reply:
Nit: For consistency, we should probably either quote the other folder ids, or unquote these.